.browserslistrc

```diff
@@ -0,0 +1,10 @@
+# The browsers that are explicitly supported
+chrome >= 57
+firefox >= 54
+safari >= 10.3
```

Comment:
Is there a way to state a different version for Safari iOS and mobile? One of them is 10.1 and the other 10.3

Reply:
Yes, there's `ios_saf`. Which version is which?

Reply:
It's Safari 10.1 and iOS Safari 10.3, so this was stricter than necessary. Will update here and in https://github.com/sumup/legacy-browsers.

Reply:
Done in 857082d.